diff -burN paragui-1.1.8.orig/src/core/pgtimerobject.cpp paragui-1.1.8/src/core/pgtimerobject.cpp
--- paragui-1.1.8.orig/src/core/pgtimerobject.cpp	2004-12-11 11:54:25.717923656 +0100
+++ paragui-1.1.8/src/core/pgtimerobject.cpp	2004-12-11 11:56:13.156590496 +0100
@@ -64,7 +64,7 @@
 }
 
 Uint32 PG_TimerObject::callbackTimer(Uint32 interval, void* data) {
-	PG_TimerObject::ID id = reinterpret_cast<PG_TimerObject::ID>(data);
+	PG_TimerObject::ID id = (PG_TimerObject::ID)(data);
 	timermap[id]->sigTimer(timermap[id], id);
 	return timermap[id]->eventTimer(id, interval);
 }
