--- paragui-1.1.8/src/widgets/pgmultilineedit.cpp~	2010-08-06 10:38:18.000000000 +0300
+++ paragui-1.1.8/src/widgets/pgmultilineedit.cpp	2010-08-06 10:38:20.371087056 +0300
@@ -99,7 +99,7 @@
 			PG_Color inv_color(255 - color.r, 255 - color.g, 255 - color.b);			
 			SetFontColor(inv_color);
 			PG_FontEngine::GetTextSize(middlepart.c_str(), GetFont(), &w);
-			SDL_Rect rect = {x + x1, y + _y, w, GetFontHeight()};
+			SDL_Rect rect = {(Sint16 )(x + x1), (Sint16)(y + _y), (Uint16 )w, (Uint16 )GetFontHeight()};
 			SDL_Surface* screen = PG_Application::GetScreen();
 			SDL_FillRect(screen, &rect, SDL_MapRGB(screen->format, color.r, color.g, color.b));
 			PG_Widget::DrawText(x1, _y, middlepart.c_str());
